articles/azure-monitor/insights/container-insights-live-logs.md

```diff
@@ -19,8 +19,10 @@ ms.author: magoedte
 Azure Monitor for containers includes a feature, which is currently in preview, that provides a live view into your Azure Kubernetes Service (AKS) container logs (stdout/stderr) and events without having to run kubectl commands. When you select either option, a new pane appears below the performance data table on the **Nodes**, **Controllers**, and **Containers** view. It shows live logging and events generated by the container engine to further assist in troubleshooting issues in real time.
 
 >[!NOTE]
->**Contributor** access to the cluster resource is required for this feature to work.
->
+>This feature is available in all public azure regions and Azure China Government Cloud (Mooncake) excluding Azure US Government Cloud (Fairfax)
```

Comment:
Mooncake and Fairfax are internally used terms and should not be included in public docs.  Suggest rewording sentence to - This feature is available in all Azure regions, including Azure China.  It is currently not available in Azure US Government.